docs/source/contributing/version-policy.md

```diff
@@ -40,10 +46,11 @@ We will always support the [latest major Plone release](https://plone.org/downlo
 
 The versions of Python that are supported in Volto depend on the version of Plone that you use.
 
-| Plone | Python       | Volto        |
-| ----- | ------------ | ------------ |
-| 6.0   | 3.8-3.11     | 16.0 or 17.0 |
-| 5.2   | 2.7, 3.6-3.8 | 15.0         |
+| Plone | Python       | Volto            |
+| ----- | ------------ | ---------------- |
+| 6.1   | 3.10-3.13    | 18.0.0           |
+| 6.0   | 3.9-3.13     | 16.0.0 or 17.0.0 |
+| 5.2   | 2.7, 3.8 .   | 15.0.0           |
 
```

Comment:
```suggestion

For Plone 6.0.x, Python 3.12 is officially supported since Plone 6.0.10.
Python 3.13 is unofficially supported since 6.0.14.
Python 3.8 support was dropped in 6.0.14.

```

Reply:
I have updated the suggestion to say that Python 3.13 is officially supported.